benchmark_how_to.md

```diff
@@ -0,0 +1,108 @@
+# Benchmarking in Activity Stream
```

Comment:
Somewhere near the top of this file, please mention that this framework is suitable for some things, and less so for others, and then link to the commentary in Benchmark.js on github that discusses this.